### PR TITLE
Add option to configure compile optimization runs

### PR DESCRIPTION
**Pull request by weilbith**
Close: #59

### CODE REVIEW

File: deploy_tools/cli.py [hunk 1]
```diff
@@ -97,6 +97,14 @@ def validate_address(ctx, param, value):
     help="Toggles the solidity optimizer",
     envvar="OPTIMIZE",
 )
+optimize_runs_option = click.option(
+    "--optimize-runs",
+    default=500,
+    type=int,
+    show_default=True,
+    help="Number of optimization runs when compile contracts",
```
Comment (berndbohmeier) [inline deploy_tools/cli.py]: This text is a little misleading. It sounds like it is the setting of how often the compiler will run the optimize algorithm. But this number is actually for how many contract runs the optimizer will optimize. 
So change it to
`Number of contract runs to optimize for when compiling contracts`

File: deploy_tools/compile.py [hunk 2]
```diff
@@ -97,6 +98,7 @@ def compile_project(
         allow_paths: Additional paths from where it is allowed to load contracts
         pattern: The pattern to find the solidity files
         optimize: Whether to turn on the solidity optimizer
+        optimize_runs: Number of runs by the solidity optimizer
```
Comment (berndbohmeier) [inline deploy_tools/compile.py]: as before, this is wrong.

File: deploy_tools/cli.py (hunk 1 above)
Comment (weilbith) [reply deploy_tools/cli.py]: When I read it again with your comment if see you point. Thanks! :smiley:

Comment (weilbith) [reply deploy_tools/cli.py]: :heavy_check_mark:

File: deploy_tools/compile.py (hunk 2 above)
Comment (weilbith) [reply deploy_tools/compile.py]: :heavy_check_mark:

**Pull request merged**